### PR TITLE
Cache user's name and last name from VACOLS 

### PR DESCRIPTION
**Pull request by odonnadieu**
Resolves https://vajira.max.gov/browse/CASEFLOW-2160

### Description
We have a nightly job (NightlySyncsJob) that creates or updates a CachedUser record which currently has sattyid, or VLJ Id. We need to add two new fields to that table: snamef and snamel (first and last name) and the job would automatically cache those fields.

### Acceptance Criteria
- [x] Code compiles correctly
- [x] CacheUser specs pass. 

### Testing Plan
Run `bundle exec rspec spec/models/cached_user_spec.rb:51` the test should pass.

### Database Changes
*Only for Schema Changes*

* [x] Have your migration classes inherit from `Caseflow::Migration`, especially when adding indexes (use `add_safe_index`)
* [x] Verify that `migrate:rollback` works as desired ([`change` supported functions](https://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method))
* [x] Perform query profiling (eyeball Rails log, check bullet and fasterer output)
* [x] Run `make docs` (after running `make migrate`) to update DB schema docs
* [x] Post this PR in #appeals-schema with a summary

### CODE REVIEW

File: db/migrate/20210812182219_add_first_and_last_name_to_cached_users.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddFirstAndLastNameToCachedUsers < Caseflow::Migration
+  def change
+    add_column :cached_user_attributes, :snamef, :string
+    add_column :cached_user_attributes, :snamel, :string
```
Comment (yoomlam) [inline db/migrate/20210812182219_add_first_and_last_name_to_cached_users.rb:4]: Add comment to describe these columns.

**Review by yoomlam — CHANGES_REQUESTED** [review]

File: db/migrate/20210812182219_add_first_and_last_name_to_cached_users.rb (hunk 1 above)
Comment (odonnadieu) [reply db/migrate/20210812182219_add_first_and_last_name_to_cached_users.rb:5]: Done ✅

Comment (yoomlam) [reply db/migrate/20210812182219_add_first_and_last_name_to_cached_users.rb:5]: I meant add comments to each column so they will appear as comments in `schema.rb` and in the DB.

Comment (odonnadieu) [reply db/migrate/20210812182219_add_first_and_last_name_to_cached_users.rb:5]:
```suggestion
class AddFirstAndLastNameToCachedUsers < Caseflow::Migration
  def change
    add_column :cached_user_attributes, :snamef, :string, comment: "User's First Name in VACOLS"
    add_column :cached_user_attributes, :snamel, :string, comment: "User's Last Name in VACOLS"
  end
end
```

Comment (odonnadieu) [reply db/migrate/20210812182219_add_first_and_last_name_to_cached_users.rb:5]: @yoomlam does the above code is the desired change?

**Review by yoomlam — APPROVED** [review]
👍

**Pull request merged**